### PR TITLE
feat: snackbar component

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**SNACKBAR / TOAST Component**

- Basic snackbar = this will pop on `top-right` of the viewport 

- Center Snackbar = This will pop on `bottom-center` of the viewport

toast msg has a bg black n color white in it .

### CODE REVIEW

File: style/components/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#snackbar {
+  background: rgb(54, 54, 54);
+  color: white;
+  padding: 15px;
+  display: inline-block;
+
+  position: fixed;
+  top: 30px;
```
Comment (Raghav888) [inline style/components/snackbar.css:8]: use rem

**Review by Raghav888 — APPROVED** [review]

File: style/components/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#snackbar {
```
Comment (JantuDeb) [inline style/components/snackbar.css:1]: Use classname instead of id, you may need this snack bar in other places also it will be an issue then.

**Review by Maianki — APPROVED** [review]
LGTM

File: docs.html [hunk 3]
```diff
@@ -613,13 +613,37 @@ <h3 class="card-title">Our Chaning Work</h3>
...
 
+  <div id="snackbar"></div>
+  <div id="snackbar-center"></div>
+  
+  <button class="btn" onclick="showMessage('Thank you for Registration !')">
+    Basic Snackbar
+  </button>
+  <button
+    class="btn"
+    onclick='showMessageCenter("Cant send photos. Retry in 5 seconds")'>Center Snackbar</button>
```
Comment (JantuDeb) [inline docs.html:636]: try to avoid using a function in HTML elements, use addEventListener() instead in js.

**Review by JantuDeb — CHANGES_REQUESTED** [review]
suggested some changes.

File: style/components/snackbar.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+#snackbar-center {
+  font-size: 80%;
+  background: rgb(54, 54, 54);
+  color: white;
+  padding: 10px 20px;
+  display: inline-block;
+
+  position: fixed;
+  bottom: 70px;
+  left: 550px;
```
Comment (arjun-computer-geek) [inline style/components/snackbar.css:24]: This is not good practice for a responsive point of view. Use rem instead px

**Review by arjun-computer-geek — CHANGES_REQUESTED** [review]

**Comment by HeyNitin** [comment]
Looks good. Just change it to rem as others have mentioned already.

File: style/components/snackbar.css (hunk 4 above)
Comment (tanishk15sharma) [reply style/components/snackbar.css:24]: sure I will change it

**Pull request merged**